### PR TITLE
add a stub for TimeTileLayer

### PR DESCRIPTION
**Pull request by cgalvarino**
@Bobfrat this should get us back up and running w/ TimeTileLayers.  For now (forever) it treats them like TileLayers.

### CODE REVIEW

**Comment by Bobfrat** [comment]
https://travis-ci.com/asascience-open/Leaflet/builds/89559871

**Comment by cgalvarino** [comment]
What am I supposed to do w/ that travis info?

**Comment by cgalvarino** [comment]
I didn't make a PR when I 1st made the oceansmap branch, so I guess that's why we didn't run into this earlier.

**Comment by Bobfrat** [comment]
> /home/travis/build/asascience-open/Leaflet/src/layer/tile/TimeTileLayer.js
  20:1  error  Expected indentation of 1 tab but found 8 spaces  indent

File: src/layer/tile/TimeTileLayer.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+export var TimeTileLayer = TileLayer.extend({
+});
+
+
+// @factory L.timeTilelayer(urlTemplate: String, options?: TimeTileLayer options)
+// Instantiates a time tile layer object given a `URL template` and optionally an options object.
+
+export function timeTileLayer(url, options) {
+        return new TimeTileLayer(url, options);
```
Comment (Bobfrat) [inline src/layer/tile/TimeTileLayer.js]: too many spaces

Comment (cgalvarino) [reply src/layer/tile/TimeTileLayer.js]: OMG.  The ghost of Luke strikes again.

**Comment by cgalvarino** [comment]
Closing in favor of #3.

**Pull request closed without merging**